test/src/process.js

```diff
@@ -3,6 +3,9 @@ import process from 'socket:process'
 import { primordials } from 'socket:ipc'
 import path from 'path-browserify'
 
+// make `path-browserify` happy
+globalThis.process = process
```

Comment:
We can also delete it from `globalThis` at the and of the module

Reply:
actually, we need `path.resolve` in the `socket:path` :)

Reply:
haha yes TODO!